src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -34,17 +34,18 @@ public class Sender : EndpointConfigurationBuilder
         {
             public Sender()
             {
-                var basePath = AppDomain.CurrentDomain.BaseDirectory;
-
-                File.WriteAllLines(Path.Combine(basePath, "DistributingACommand.Receiver.txt"), new[]
-                {
-                    "1:",
-                    "2:"
-                });
+                var filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "routes.xml");
+                File.WriteAllText(filePath, @"<endpoints>
+    <endpoint name=""DistributingACommand.Receiver"">
+        <instance scaleOutDiscriminator=""1""/>
```

Comment:
OK guys, I am on the fence here:
 * @andreasohlund wants `discriminator`
 * @tmasternak wants `scaleOutDiscriminator`
 * @timbussmann wants `instanceId`
I'll change it back to discriminator not because I prefer what @andreasohlund says but because this is what it was called before. If you guys feel like it needs to be changed, please propose another PR. I don't want this rename to block the whole PR.